target_chains/cosmwasm/tools/src/configs.ts

```diff
@@ -0,0 +1,233 @@
+};
+
+const MainnetContractConfig: Record<ChainIdMainnet, ContractConfig> = {
+  [ChainIdMainnet.INJECTIVE]: {
+    feeDenom: "inj",
+    pythArtifactZipName: "injective",
+    wormholeChainId: CHAINS.injective,
+  },
+  [ChainIdMainnet.OSMOSIS]: {
+    feeDenom: "uosmo",
+    pythArtifactZipName: "osmosis",
+    wormholeChainId: CHAINS.osmosis,
+  },
+};
+
+const TestnetEdgeContractConfig: Record<ChainIdTestnet, ContractConfig> = {
+  [ChainIdTestnet.INJECTIVE]: {
+    feeDenom: "inj",
+    pythArtifactZipName: "injective",
+    wormholeChainId: CHAINS.injective,
+  },
+  [ChainIdTestnet.OSMOSIS_4]: {
+    feeDenom: "uosmo",
+    pythArtifactZipName: "osmosis",
+    wormholeChainId: CHAINS.osmosis,
+  },
+  [ChainIdTestnet.OSMOSIS_5]: {
+    feeDenom: "uosmo",
+    pythArtifactZipName: "osmosis",
+    wormholeChainId: CHAINS.osmosis,
+  },
+  [ChainIdTestnet.SEI_ATLANTIC_2]: {
+    feeDenom: "usei",
+    pythArtifactZipName: "cosmwasm",
+    wormholeChainId: CHAINS.sei,
+  },
+  [ChainIdTestnet.NEUTRON_PION_1]: {
+    feeDenom: "untrn",
+    pythArtifactZipName: "cosmwasm",
+    wormholeChainId: CHAINS.neutron,
+  },
+  [ChainIdTestnet.JUNO]: {
+    feeDenom: "ujunox",
+    pythArtifactZipName: "cosmwasm",
+    wormholeChainId: CHAINS.juno,
+  },
+};
+
+const TestnetStableContractConfig: Record<ChainIdTestnet, ContractConfig> = {
+  [ChainIdTestnet.INJECTIVE]: {
+    feeDenom: "inj",
+    pythArtifactZipName: "injective",
+    wormholeChainId: CHAINS.injective_stable,
+  },
+  [ChainIdTestnet.OSMOSIS_4]: {
+    feeDenom: "uosmo",
+    pythArtifactZipName: "osmosis",
+    wormholeChainId: CHAINS.osmosis_stable,
+  },
+  [ChainIdTestnet.OSMOSIS_5]: {
+    feeDenom: "uosmo",
+    pythArtifactZipName: "osmosis",
+    wormholeChainId: CHAINS.osmosis_stable,
+  },
+  [ChainIdTestnet.SEI_ATLANTIC_2]: {
+    feeDenom: "usei",
+    pythArtifactZipName: "cosmwasm",
+    wormholeChainId: CHAINS.sei_stable,
+  },
+  [ChainIdTestnet.NEUTRON_PION_1]: {
+    feeDenom: "untrn",
+    pythArtifactZipName: "cosmwasm",
+    wormholeChainId: CHAINS.neutron_stable,
+  },
+  [ChainIdTestnet.JUNO]: {
+    feeDenom: "ujunox",
+    pythArtifactZipName: "cosmwasm",
+    wormholeChainId: CHAINS.juno_stable,
+  },
+};
+
+export function getContractConfig(
+  chainId: ChainIdMainnet | ChainIdTestnet,
+  deploymentType: DeploymentType
+): ContractConfig {
+  if (deploymentType === "mainnet")
+    return MainnetContractConfig[chainId as ChainIdMainnet];
+  else if (deploymentType === "testnet-stable")
+    return TestnetStableContractConfig[chainId as ChainIdTestnet];
+
+  // testnet-edge
+  return TestnetEdgeContractConfig[chainId as ChainIdTestnet];
+}
+
+export function getChainConfig(
+  chainId: ChainIdTestnet | ChainIdMainnet,
+  deploymentType: DeploymentType
+) {
+  if (deploymentType === "mainnet")
+    return ChainsConfigMainnet[chainId as ChainIdMainnet];
+  return ChainsConfigTestnet[chainId as ChainIdTestnet];
+}
+
+function getPythSources(mainnet: boolean) {
```

Comment:
isn't `mainnet` now the distinction between stable vs. edge? We should give this variable a different name to clarify that

Reply:
maybe use DeploymentType?